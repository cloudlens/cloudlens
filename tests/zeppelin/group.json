{"paragraphs":[{"text":"%cl\nsource(\"local:tests/zeppelin/group.log\")","dateUpdated":"Jun 27, 2016 3:45:56 PM","config":{"colWidth":12,"graph":{"mode":"table","height":279,"optionOpen":false,"keys":[],"values":[],"groups":[],"scatter":{}},"enabled":true,"editorMode":"ace/mode/scala"},"settings":{"params":{},"forms":{}},"jobName":"paragraph_1467056745445_1007485465","id":"20160627-154545_1387718685","result":{"code":"SUCCESS","type":"TABLE","msg":"\n%html <font color=blue>message</font>:&quot;Alice picked:&quot;\t\n%html <font color=blue>message</font>:&quot;3 apples&quot;\t\n%html <font color=blue>message</font>:&quot;Nicolas picked more: &quot;\t\n%html <font color=blue>message</font>:&quot;4 apples&quot;\t\n%html <font color=blue>message</font>:&quot;Mandana picked:&quot;\t\n%html <font color=blue>message</font>:&quot;6 oranges&quot;\t\n%html <font color=blue>message</font>:&quot;Olivier picked the most: &quot;\t\n%html <font color=blue>message</font>:&quot;5 apples&quot;\t","comment":"","msgTable":[[{"value":"%html <font color=blue>message</font>:&quot;Alice picked:&quot;"},{"value":""}],[{"value":"%html <font color=blue>message</font>:&quot;3 apples&quot;"},{"value":""}],[{"value":"%html <font color=blue>message</font>:&quot;Nicolas picked more: &quot;"},{"value":""}],[{"value":"%html <font color=blue>message</font>:&quot;4 apples&quot;"},{"value":""}],[{"value":"%html <font color=blue>message</font>:&quot;Mandana picked:&quot;"},{"value":""}],[{"value":"%html <font color=blue>message</font>:&quot;6 oranges&quot;"},{"value":""}],[{"value":"%html <font color=blue>message</font>:&quot;Olivier picked the most: &quot;"},{"value":""}],[{"value":"%html <font color=blue>message</font>:&quot;5 apples&quot;"},{"value":""}]],"columnNames":[],"rows":[["%html <font color=blue>message</font>:&quot;Alice picked:&quot;",""],["%html <font color=blue>message</font>:&quot;3 apples&quot;",""],["%html <font color=blue>message</font>:&quot;Nicolas picked more: &quot;",""],["%html <font color=blue>message</font>:&quot;4 apples&quot;",""],["%html <font color=blue>message</font>:&quot;Mandana picked:&quot;",""],["%html <font color=blue>message</font>:&quot;6 oranges&quot;",""],["%html <font color=blue>message</font>:&quot;Olivier picked the most: &quot;",""],["%html <font color=blue>message</font>:&quot;5 apples&quot;",""]]},"dateCreated":"Jun 27, 2016 3:45:45 PM","dateStarted":"Jun 27, 2016 3:45:57 PM","dateFinished":"Jun 27, 2016 3:45:57 PM","status":"FINISHED","progressUpdateIntervalMs":500,"$$hashKey":"object:678"},{"text":"%cl\ngroup{\n    \"^[^\\s]+ picked\"\n}","dateUpdated":"Jul 2, 2016 10:07:39 PM","config":{"colWidth":12,"graph":{"mode":"table","height":300,"optionOpen":false,"keys":[],"values":[],"groups":[],"scatter":{}},"enabled":true,"editorMode":"ace/mode/scala"},"settings":{"params":{},"forms":{}},"jobName":"paragraph_1467056745445_1007485465","id":"20160627-154545_477123389","result":{"code":"SUCCESS","type":"TABLE","msg":"\n%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Alice picked:&quot;},&quot;1&quot;:{&quot;message&quot;:&quot;3 apples&quot;}}\t\n%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Nicolas picked more: &quot;},&quot;1&quot;:{&quot;message&quot;:&quot;4 apples&quot;}}\t\n%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Mandana picked:&quot;},&quot;1&quot;:{&quot;message&quot;:&quot;6 oranges&quot;}}\t\n%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Olivier picked the most: &quot;},&quot;1&quot;:{&quot;message&quot;:&quot;5 apples&quot;}}\t","comment":"","msgTable":[[{"value":"%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Alice picked:&quot;},&quot;1&quot;:{&quot;message&quot;:&quot;3 apples&quot;}}"},{"value":""}],[{"value":"%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Nicolas picked more: &quot;},&quot;1&quot;:{&quot;message&quot;:&quot;4 apples&quot;}}"},{"value":""}],[{"value":"%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Mandana picked:&quot;},&quot;1&quot;:{&quot;message&quot;:&quot;6 oranges&quot;}}"},{"value":""}],[{"value":"%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Olivier picked the most: &quot;},&quot;1&quot;:{&quot;message&quot;:&quot;5 apples&quot;}}"},{"value":""}]],"columnNames":[],"rows":[["%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Alice picked:&quot;},&quot;1&quot;:{&quot;message&quot;:&quot;3 apples&quot;}}",""],["%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Nicolas picked more: &quot;},&quot;1&quot;:{&quot;message&quot;:&quot;4 apples&quot;}}",""],["%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Mandana picked:&quot;},&quot;1&quot;:{&quot;message&quot;:&quot;6 oranges&quot;}}",""],["%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Olivier picked the most: &quot;},&quot;1&quot;:{&quot;message&quot;:&quot;5 apples&quot;}}",""]]},"dateCreated":"Jun 27, 2016 3:45:45 PM","dateStarted":"Jun 27, 2016 3:46:06 PM","dateFinished":"Jun 27, 2016 3:46:06 PM","status":"FINISHED","progressUpdateIntervalMs":500,"$$hashKey":"object:679"},{"text":"%cl\nstream(entry) when (entry){\n    entry.message = \"\";\n    for (var line in entry.array){\n        entry.message += entry.array[line].message;\n    }\n}\n","dateUpdated":"Jun 27, 2016 3:46:49 PM","config":{"colWidth":12,"graph":{"mode":"table","height":300,"optionOpen":false,"keys":[],"values":[],"groups":[],"scatter":{}},"enabled":true,"editorMode":"ace/mode/scala"},"settings":{"params":{},"forms":{}},"jobName":"paragraph_1467056745445_1007485465","id":"20160627-154545_825986519","result":{"code":"SUCCESS","type":"TABLE","msg":"\n%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Alice picked:&quot;},&quot;1&quot;:{&quot;message&quot;:&quot;3 apples&quot;}}\t%html <font color=blue>message</font>:&quot;Alice picked:3 apples&quot;\t\n%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Nicolas picked more: &quot;},&quot;1&quot;:{&quot;message&quot;:&quot;4 apples&quot;}}\t%html <font color=blue>message</font>:&quot;Nicolas picked more: 4 apples&quot;\t\n%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Mandana picked:&quot;},&quot;1&quot;:{&quot;message&quot;:&quot;6 oranges&quot;}}\t%html <font color=blue>message</font>:&quot;Mandana picked:6 oranges&quot;\t\n%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Olivier picked the most: &quot;},&quot;1&quot;:{&quot;message&quot;:&quot;5 apples&quot;}}\t%html <font color=blue>message</font>:&quot;Olivier picked the most: 5 apples&quot;\t","comment":"","msgTable":[[{"value":"%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Alice picked:&quot;},&quot;1&quot;:{&quot;message&quot;:&quot;3 apples&quot;}}"},{"value":"%html <font color=blue>message</font>:&quot;Alice picked:3 apples&quot;"},{"value":""}],[{"value":"%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Nicolas picked more: &quot;},&quot;1&quot;:{&quot;message&quot;:&quot;4 apples&quot;}}"},{"value":"%html <font color=blue>message</font>:&quot;Nicolas picked more: 4 apples&quot;"},{"value":""}],[{"value":"%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Mandana picked:&quot;},&quot;1&quot;:{&quot;message&quot;:&quot;6 oranges&quot;}}"},{"value":"%html <font color=blue>message</font>:&quot;Mandana picked:6 oranges&quot;"},{"value":""}],[{"value":"%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Olivier picked the most: &quot;},&quot;1&quot;:{&quot;message&quot;:&quot;5 apples&quot;}}"},{"value":"%html <font color=blue>message</font>:&quot;Olivier picked the most: 5 apples&quot;"},{"value":""}]],"columnNames":[],"rows":[["%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Alice picked:&quot;},&quot;1&quot;:{&quot;message&quot;:&quot;3 apples&quot;}}","%html <font color=blue>message</font>:&quot;Alice picked:3 apples&quot;",""],["%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Nicolas picked more: &quot;},&quot;1&quot;:{&quot;message&quot;:&quot;4 apples&quot;}}","%html <font color=blue>message</font>:&quot;Nicolas picked more: 4 apples&quot;",""],["%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Mandana picked:&quot;},&quot;1&quot;:{&quot;message&quot;:&quot;6 oranges&quot;}}","%html <font color=blue>message</font>:&quot;Mandana picked:6 oranges&quot;",""],["%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Olivier picked the most: &quot;},&quot;1&quot;:{&quot;message&quot;:&quot;5 apples&quot;}}","%html <font color=blue>message</font>:&quot;Olivier picked the most: 5 apples&quot;",""]]},"dateCreated":"Jun 27, 2016 3:45:45 PM","dateStarted":"Jun 27, 2016 3:46:49 PM","dateFinished":"Jun 27, 2016 3:46:49 PM","status":"FINISHED","progressUpdateIntervalMs":500,"$$hashKey":"object:680"},{"text":"%cl\nmatch{\n    \"(?<numApples>\\d+) apples\";\n    \"^(?<name>[^\\s]*) picked\"\n}","dateUpdated":"Jul 2, 2016 10:07:40 PM","config":{"colWidth":12,"graph":{"mode":"table","height":300,"optionOpen":false,"keys":[],"values":[],"groups":[],"scatter":{}},"enabled":true,"editorMode":"ace/mode/scala"},"settings":{"params":{},"forms":{}},"jobName":"paragraph_1467056745445_1007485465","id":"20160627-154545_183014008","result":{"code":"SUCCESS","type":"TABLE","msg":"\n%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Alice picked:&quot;},&quot;1&quot;:{&quot;message&quot;:&quot;3 apples&quot;}}\t%html <font color=blue>message</font>:&quot;Alice picked:3 apples&quot;\t%html <font color=blue>numApples</font>:&quot;3&quot;\t%html <font color=blue>name</font>:&quot;Alice&quot;\t\n%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Nicolas picked more: &quot;},&quot;1&quot;:{&quot;message&quot;:&quot;4 apples&quot;}}\t%html <font color=blue>message</font>:&quot;Nicolas picked more: 4 apples&quot;\t%html <font color=blue>numApples</font>:&quot;4&quot;\t%html <font color=blue>name</font>:&quot;Nicolas&quot;\t\n%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Mandana picked:&quot;},&quot;1&quot;:{&quot;message&quot;:&quot;6 oranges&quot;}}\t%html <font color=blue>message</font>:&quot;Mandana picked:6 oranges&quot;\t%html <font color=blue>name</font>:&quot;Mandana&quot;\t\n%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Olivier picked the most: &quot;},&quot;1&quot;:{&quot;message&quot;:&quot;5 apples&quot;}}\t%html <font color=blue>message</font>:&quot;Olivier picked the most: 5 apples&quot;\t%html <font color=blue>numApples</font>:&quot;5&quot;\t%html <font color=blue>name</font>:&quot;Olivier&quot;\t","comment":"","msgTable":[[{"value":"%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Alice picked:&quot;},&quot;1&quot;:{&quot;message&quot;:&quot;3 apples&quot;}}"},{"value":"%html <font color=blue>message</font>:&quot;Alice picked:3 apples&quot;"},{"value":"%html <font color=blue>numApples</font>:&quot;3&quot;"},{"value":"%html <font color=blue>name</font>:&quot;Alice&quot;"},{"value":""}],[{"value":"%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Nicolas picked more: &quot;},&quot;1&quot;:{&quot;message&quot;:&quot;4 apples&quot;}}"},{"value":"%html <font color=blue>message</font>:&quot;Nicolas picked more: 4 apples&quot;"},{"value":"%html <font color=blue>numApples</font>:&quot;4&quot;"},{"value":"%html <font color=blue>name</font>:&quot;Nicolas&quot;"},{"value":""}],[{"value":"%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Mandana picked:&quot;},&quot;1&quot;:{&quot;message&quot;:&quot;6 oranges&quot;}}"},{"value":"%html <font color=blue>message</font>:&quot;Mandana picked:6 oranges&quot;"},{"value":"%html <font color=blue>name</font>:&quot;Mandana&quot;"},{"value":""}],[{"value":"%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Olivier picked the most: &quot;},&quot;1&quot;:{&quot;message&quot;:&quot;5 apples&quot;}}"},{"value":"%html <font color=blue>message</font>:&quot;Olivier picked the most: 5 apples&quot;"},{"value":"%html <font color=blue>numApples</font>:&quot;5&quot;"},{"value":"%html <font color=blue>name</font>:&quot;Olivier&quot;"},{"value":""}]],"columnNames":[],"rows":[["%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Alice picked:&quot;},&quot;1&quot;:{&quot;message&quot;:&quot;3 apples&quot;}}","%html <font color=blue>message</font>:&quot;Alice picked:3 apples&quot;","%html <font color=blue>numApples</font>:&quot;3&quot;","%html <font color=blue>name</font>:&quot;Alice&quot;",""],["%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Nicolas picked more: &quot;},&quot;1&quot;:{&quot;message&quot;:&quot;4 apples&quot;}}","%html <font color=blue>message</font>:&quot;Nicolas picked more: 4 apples&quot;","%html <font color=blue>numApples</font>:&quot;4&quot;","%html <font color=blue>name</font>:&quot;Nicolas&quot;",""],["%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Mandana picked:&quot;},&quot;1&quot;:{&quot;message&quot;:&quot;6 oranges&quot;}}","%html <font color=blue>message</font>:&quot;Mandana picked:6 oranges&quot;","%html <font color=blue>name</font>:&quot;Mandana&quot;",""],["%html <font color=blue>group</font>:{&quot;0&quot;:{&quot;message&quot;:&quot;Olivier picked the most: &quot;},&quot;1&quot;:{&quot;message&quot;:&quot;5 apples&quot;}}","%html <font color=blue>message</font>:&quot;Olivier picked the most: 5 apples&quot;","%html <font color=blue>numApples</font>:&quot;5&quot;","%html <font color=blue>name</font>:&quot;Olivier&quot;",""]]},"dateCreated":"Jun 27, 2016 3:45:45 PM","dateStarted":"Jun 27, 2016 3:47:14 PM","dateFinished":"Jun 27, 2016 3:47:14 PM","status":"FINISHED","progressUpdateIntervalMs":500,"$$hashKey":"object:681"},{"text":"%cl\nstream when (entry.numApples){\n    print(entry.name + \" picked \" + entry.numApples)\n}","dateUpdated":"Jun 27, 2016 3:47:26 PM","config":{"colWidth":12,"graph":{"mode":"table","height":300,"optionOpen":false,"keys":[],"values":[],"groups":[],"scatter":{}},"enabled":true,"editorMode":"ace/mode/scala"},"settings":{"params":{},"forms":{}},"jobName":"paragraph_1467056745445_1007485465","id":"20160627-154545_1878746424","result":{"code":"SUCCESS","type":"TEXT","msg":"Alice picked 3\nNicolas picked 4\nOlivier picked 5\n"},"dateCreated":"Jun 27, 2016 3:45:45 PM","dateStarted":"Jun 27, 2016 3:47:26 PM","dateFinished":"Jun 27, 2016 3:47:26 PM","status":"FINISHED","progressUpdateIntervalMs":500,"$$hashKey":"object:682"},{"dateUpdated":"Jun 27, 2016 3:45:45 PM","config":{"colWidth":12,"graph":{"mode":"table","height":300,"optionOpen":false,"keys":[],"values":[],"groups":[],"scatter":{}},"enabled":true,"editorMode":"ace/mode/scala"},"settings":{"params":{},"forms":{}},"jobName":"paragraph_1467056745445_1007485465","id":"20160627-154545_1905011265","dateCreated":"Jun 27, 2016 3:45:45 PM","status":"READY","progressUpdateIntervalMs":500,"$$hashKey":"object:683"}],"name":"Multi-line Log Entries Example","id":"2BPAHRTNX","angularObjects":{"2BRB4ZK9Z":[],"2BN98KZFM":[]},"config":{"looknfeel":"default"},"info":{}}
