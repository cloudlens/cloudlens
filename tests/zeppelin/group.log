Alice picked:
3 apples
Nicolas picked more: 
4 apples
Mandana picked:
6 oranges
Olivier picked the most: 
5 apples
