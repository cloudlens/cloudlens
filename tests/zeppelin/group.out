Alice picked 3
Nicolas picked 4
Olivier picked 5
